README.md

```diff
@@ -13,18 +13,18 @@ Start using it straight away. (Following example assumes that it was installed v
 ```php
 <?php
 
-require 'path/to/vendor/autoload.php';
+require __DIR__.'/vendor/autoload.php';
```

Comment:
No problem and glad to help! I just added it back instead of trying to get rid of that commit.